modules/siddhi-core/src/main/java/io/siddhi/core/table/record/RecordStoreCompiledCondition.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Let's change `2016` to `2020`

Reply:
deleted class and moved it inside AbstractRecordTable